dev/devicelab/manifest.yaml

```diff
@@ -146,6 +146,20 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf_ios__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    flaky: true
+    required_agent_capabilities: ["mac/ios"]
```

Comment:
What's the difference between ios and iphonex? It's confusing to see both of them.

Reply:
"mac/ios" is iPhone 6, "mac/ios32" is iPhone 4s.

I'm adding the device model to all ios task descriptions.